<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<<<<<<< HEAD
<testsuite ignored="0" hostname="Udayas-Air" failures="1" tests="3" name="Default test" time="82.842" errors="0" timestamp="2023-05-26T08:44:49 IST">
  <testcase classname="pageTests.YourCartPageTest" name="verifyTitle_YourCartPage" time="6.227"/>
  <testcase classname="pageTests.YourCartPageTest" name="verify_Product_In_Cart" time="3.361"/>
  <testcase classname="pageTests.YourCartPageTest" name="verify_removeProduct_From_CartPage" time="13.788">
    <failure type="java.lang.AssertionError" message="Product from cart is not removed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Product from cart is not removed expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:819)
at pageTests.YourCartPageTest.verify_removeProduct_From_CartPage(YourCartPageTest.java:108)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verify_removeProduct_From_CartPage -->
=======
<testsuite ignored="2" hostname="UTIPPANAMBP18.local" failures="0" tests="1" name="Default test" time="18.208" errors="0" timestamp="2023-05-22T14:55:21 IST">
  <testcase classname="pageTests.YourCartPageTest" name="verify_Product_In_Cart" time="5.924"/>
  <testcase classname="pageTests.YourCartPageTest" name="verify_removeProduct_From_CartPage" time="0.0">
    <ignored/>
  </testcase> <!-- verify_removeProduct_From_CartPage -->
  <testcase classname="pageTests.YourCartPageTest" name="verifyTitle_YourCartPage" time="0.0">
    <ignored/>
  </testcase> <!-- verifyTitle_YourCartPage -->
>>>>>>> f5175878247a5f893a0168f1ba8f5f385cd2c0cd
</testsuite> <!-- Default test -->
